homeassistant/components/keba/binary_sensor.py

```diff
@@ -0,0 +1,110 @@
+"""Support for KEBA charging station binary sensors."""
+import logging
+
+from homeassistant.components.binary_sensor import BinarySensorDevice
+
+from . import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_platform(hass, config,
+                               async_add_entities, discovery_info=None):
+    """Set up the KEBA charging station platform."""
+    _LOGGER.debug("Initializing KEBA charging station binary sensors")
+    keba = hass.data[DOMAIN]
+
+    sensors = [
+        KebaBinarySensor(keba, 'Online', 'Wallbox', 'connectifity'),
```

Comment:
Import the device class constants from the base binary_sensor component and use here.

Reply:
done